docs/quickstarts/sso-quickstart.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Use the Yeoman generator to create an Office Add-in that uses SSO (preview)
+description: Use the Yeoman generator to build a Node.js Office Add-in that uses single sign-on (preview).
+ms.date: 01/13/2020
+ms.prod: non-product-specific
+localization_priority: Priority
+---
+
+# Use the Yeoman generator to create an Office Add-in that uses single sign-on (preview)
+
+In this article, you'll walk through the process of using the Yeoman generator to create an Office Add-in for Excel, Word, or PowerPoint that uses single sign-on (SSO) when possible, and uses an alternate method of user authentication when SSO is not supported.
+
+> [!TIP]
+> Before you attempt to complete this quick start, review [Enable single sign-on for Office Add-ins](../develop/sso-in-office-add-ins.md) to learn basic concepts about SSO in Office Add-ins. 
+ 
+The Yeoman generator simplifies the process of creating an SSO add-in, by automating the steps required to configure SSO within Azure and generating the code that's necessary for an add-in to use SSO. For a detailed walkthrough that describes how to manually complete the steps that the Yeoman generator automates, see the [Create a Node.js Office Add-in that uses single sign-on](../develop/create-sso-office-add-ins-nodejs.md) tutorial.
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org) (version 10.15.0 or later)
+
+- The latest version of [Yeoman](https://github.com/yeoman/yo) and the [Yeoman generator for Office Add-ins](https://github.com/OfficeDev/generator-office). To install these tools globally, run the following command via the command prompt:
+
+    ```command&nbsp;line
+    npm install -g yo generator-office
+    ```
+
+    [!include[note to update Yeoman generator](../includes/note-yeoman-generator-update.md)]
+
+- An Office 365 (the subscription version of Office) account, which you can get by joining the [Office 365 Developer Program](https://aka.ms/devprogramsignup) that includes a free 1 year subscription to Office 365. You should use the latest monthly version and build from the Insiders channel but you need to be an Office Insider to get this version. For more information, see [Be an Office Insider](https://products.office.com/office-insider?tab=tab-1). Please note that when a build graduates to the production semi-annual channel, support for preview features, including SSO, is turned off for that build.
```

Comment:
There is a lot of information in this bullet point. Can we 
1) start with "An Insider's build of Office 365..."? The first time I read the quickstart, I skimmed this and completely missed that I needed to be on the Insider version of my Office subscription
2) separate the "Please note..." section into a note below the bullet point?

Reply:
@nokafor valid point and good suggestions. 

FWIW, I actually copied this bullet point (in entirety) from Ricky's [Node.js tutorial](https://review.docs.microsoft.com/en-us/office/dev/add-ins/develop/create-sso-office-add-ins-nodejs?branch=kb-sso-quickstart) which has existed for a while now. I'll update it here in the SSO quick start and will create a VSO work item for @Rick-Kirkham to evaluate updating similarly his Node.js tutorial (and anywhere else it may be included in the SSO-related docs).